### PR TITLE
InvalidDirectory cannot be resolved in this context

### PR DESCRIPTION
**Pull request by unixcharles**
fix https://github.com/unixcharles/acme-client/issues/142

### CODE REVIEW

**Comment by jeromegn** [comment]
Apart from the rubocop offense, this lgtm.

**Comment by jodosha** [comment]
👋 Thanks for this PR. Is there any chance to merge and release this fix? We're starting to see exceptions in production due to https://github.com/unixcharles/acme-client/issues/142.

If there is anything that I can do to help in the process, please let me know.

TYVM for `acme-client` BTW! 💚

File: lib/acme/client/resources/directory.rb [hunk 1]
```diff
@@ -63,7 +63,7 @@ def load_directory
     end
     result
   rescue JSON::ParserError => exception
-    raise InvalidDirectory, "Invalid directory url\n#{@directory} did not return a valid directory\n#{exception.inspect}"
+    raise Acme::Client::Error::InvalidDirectory, "Invalid directory url\n#{@directory} did not return a valid directory\n#{exception.inspect}"
```
Comment (weppos) [inline lib/acme/client/resources/directory.rb]: To please rubocop:

```suggestion
    raise Acme::Client::Error::InvalidDirectory,
        "Invalid directory url\n#{@directory} did not return a valid directory\n#{exception.inspect}"
```

**Comment by unixcharles** [comment]
Sorry about that @jodosha, completely forgot about that. I'll merge a release a new version.

**Pull request merged**